google_auth_oauthlib/flow.py

```diff
@@ -229,10 +235,9 @@ def fetch_token(self, **kwargs):
                 :meth:`credentials` to obtain a
                 :class:`~google.auth.credentials.Credentials` instance.
         """
+        kwargs.setdefault('client_secret', self.client_config['client_secret'])
```

Comment:
`KeyError` ouch?

Reply:
It's okay if that happens (for now at least)

Reply:
:+1: